base_agent/dialogue_objects/filter_helper.py

```diff
@@ -53,6 +53,125 @@ def get_pos(self):
     return selector
 
 
+def get_val_map(interpreter, speaker, filters_d, get_all=False):
+    output = filters_d.get("output")
+    val_map = None
+    if output and type(output) is dict:
+        attr_d = output.get("attribute")
+        get_attribute = interpreter.subinterpret.get("attribute", AttributeInterpreter())
+        a = get_attribute(interpreter, speaker, attr_d, get_all=get_all)
+        val_map = ApplyAttribute(interpreter.agent.memory, a)
+    elif output and output == "count":
+        val_map = CountTransform(interpreter.agent.memory)
+    return val_map
```

Comment:
then this will output None, and farther down
https://github.com/facebookresearch/droidlet/blob/812c4f63b7e0804a6aa44ed206de9445459240e9/base_agent/dialogue_objects/filter_helper.py#L206
nothing will be appended to the head of the FILTER, and so it will just output the memory